Makefile

```diff
@@ -206,6 +206,7 @@ get-deps: get-deps-tests get-deps-x-tests get-deps-codegen get-deps-verify
 get-deps-tests:
 	@echo "go get SDK testing dependencies"
 	go get github.com/stretchr/testify
+	go get github.com/google/go-cmp/...
```

Comment:
is the nested `/...` of this needed?

Reply:
We can either have gocmp/... or point directly to gocmp/cmp pkg instead. I think the later would be better.